app/pages/cif/projects.tsx

```diff
@@ -118,7 +118,7 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
         "Primary Project Managers",
         "primaryProjectManager",
         allCifUsers.edges.map((e) => e.node.fullName),
-        { allowFreeFormInput: true, sortable: false }
+        { allowFreeFormInput: false, sortable: false }
```

Comment:
We can simply remove whole `allowFreeFormInput: false` as it's false by default.

Reply:
Good catch! Thank you